### PR TITLE
Create new trigger for EGun.

### PR DESCRIPTION
**Pull request by fernandohds564**

### CODE REVIEW

**Review by murilobalves — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Comment by anacso17** [comment]
Hi guys, @lerwys @danielot , could you take a look at this PR?

File: timesys/high-level-triggers.py [hunk 1]
```diff
@@ -696,10 +696,22 @@
             'NrPulses': {'value': 1, 'hilim': 2, 'high': 2, 'hihi': 2},
             'Duration': {'value': 0.3, 'hilim': 1.6, 'high': 1.6, 'hihi': 1.6},
             'State': {'value': 0},
-            'Polarity': {'value': 1, 'states': (0, 2)},
+            'Polarity': {'value': 0, 'states': (0, 2)},
```
Comment (lerwys) [inline timesys/high-level-triggers.py:699]: Silly question. Is the polarity value correct here?

Comment (anacso17) [reply timesys/high-level-triggers.py:699]: Looking at the last saved config, LI-01:TI-EGun-MultBun:Polarity-Sel and LI-01: TI-EGun-SglBun: Polarity-Sel are 0 indeed, so I thing it is correct!

Comment (lerwys) [reply timesys/high-level-triggers.py:699]: Ok then. Nice.

**Review by lerwys — APPROVED** [review]

**Pull request merged**